### PR TITLE
[exu] Add chicken bit to kill performance (AKA "Single stage in order pipeline support in a superscalar out-of-order core")

### PR DESCRIPTION
**Pull request by jerryz123**
**Related issue**: 

**Type of change**: new feature

**Impact**: new rtl

**Development Phase**: implementation

**Release Notes**

`csrwi 0x7c1, 0x8` will disable out-of-orderiness in BOOM. All instructions will wait for pipeline to drain before issue.

### CODE REVIEW

**Comment by abejgonzalez** [comment]
I request that the name of the PR change to "Single stage in order pipeline support in a superscalar out-of-order core"

**Comment by ccelio** [comment]
Incoming "hybrid/morph core" ISCA paper...

File: docs/sections/debugging.rst [hunk 1]
```diff
@@ -9,6 +9,13 @@ the FireSim tool to debug faster using an FPGA. This tools comes out of the
 UC Berkeley Architecture Research group and is still a work in progress. You
 can find the documentation and website at https://fires.im/.
 
+Chicken Bits
+------------
+BOOM supports a chicken-bit to delay all instructions from issue until the
```
Comment (abejgonzalez) [inline docs/sections/debugging.rst:14]: I would also mention that this is able to be toggled dynamically at runtime (so you could potentially run particular sections of the code with it on/off)

**Review by abejgonzalez — APPROVED** [review]
LGTM with the small comment change.

**Comment by abejgonzalez** [comment]
We should add the security stuff and call it the BOOM + Rocket + Security = "Block-et" core.

**Pull request merged**